src/main/java/jpo/sdw/depositor/depositors/SDWDepositor.java

```diff
@@ -29,7 +29,7 @@ public SDWDepositor(DepositorProperties props, JavaMailSender sender, WebClient
 
    @Override
    public void deposit(String message) {
-      Mono<ClientResponse> clientResponse = this.getWebClient().post().body(BodyInserters.fromObject(message))
+      Mono<ClientResponse> clientResponse = this.getWebClient().post().header("apiKey", depositorProperties.getApiKey()).body(BodyInserters.fromValue(message))
```

Comment:
The `apiKey` should already be a part of the webClient default headers set in DepositController.java, I don't think we need to add it here.

Reply:
Removed the header and checked that the tests still work